support.js

```diff
@@ -303,8 +304,9 @@ if (async.length) {
 }
 
 function onLoaded(fn) {
-  if (document.readyState == "loading") {
-    document.addEventListener('DOMContentLoaded', fn);
+  if (document.readyState == "loading" ||
+      document.readyState == "interactive") {
+    window.addEventListener('load', fn);
```

Comment:
I'm guessing IE9 doesn't have DOMContentLoaded either?  A comment would be nice to remind me not to change it back when I stumble across this later.